core/txpool/blobpool/blobpool.go

```diff
@@ -335,7 +335,7 @@ func New(config Config, chain BlockChain) *BlobPool {
 		index:  make(map[common.Address][]*blobTxMeta),
 		spent:  make(map[common.Address]*uint256.Int),
 
-		feeCurrencyValidator: txpool.NewFeeCurrencyValidator(),
+		feeCurrencyValidator: txpool.NewFeeCurrencyValidator(chain.Config()),
```

Comment:
I couldn't find where `feeCurrencyValidator` is used in blobpool.

Reply:
in validateTx()

Reply:
although that's not good enough reason as to have it as a member, I'll revert that